### ISSUE TITLE
Regression 0.4.0: "Error converting __.svg into ___.png" with GitBook

### ISSUE THREAD

**Issue by bkimminich** [shakiba/svgexport#71]
# Regression

Without any relevant changes on my end an AppVeyor job I have running GitBook CLI fails when using `0.4.0` while trying to convert an SVG into a PNG:

## ✔️ https://ci.appveyor.com/project/bkimminich/pwning-juice-shop/builds/30108321
(before release of `0.4.0`, dependency resolving to `0.3.2`)

## ❌ https://ci.appveyor.com/project/bkimminich/pwning-juice-shop/builds/30265687
(since release of `0.4.0`)
```
error: error while generating page "part1/running.md": 
Error: Error converting C:\Users\appveyor\AppData\Local\Temp\tmp-2028DZAryLK1LD5K\part1\img\deploy-to-heroku.svg into C:\Users\appveyor\AppData\Local\Temp\tmp-2028DZAryLK1LD5K\30c8bde3.png
Command exited with code 1
```

## ✔️ https://ci.appveyor.com/project/bkimminich/pwning-juice-shop/builds/30271052
(after pinning to `0.3.2`)

**Comment by shakiba** [comment]
Yup, we have recently update from PhantomJS to Puppeteer. @MichaelHeerklotz is leading the work, so he might be more helpful.

**Comment by MichaelHeerklotz** [comment]
There seem to be some problems when running on Windows. I am working on a fix.

**Comment by greyltc** [comment]
I just ran into this today in debian. @MichaelRoosz any word on that fix?

**Comment by henryiii** [comment]
In https://gitlab.com/cliutils/modern-cmake, I'm hitting this as of a week or two ago, and it's not tied to 0.3/0.4, I tried pinning the older one and it didn't fix it, and newer ones used to be fine. I think an underlying library must have changed. Any idea of what it might be, or how to see a more informative error message?

**Comment by henryiii** [comment]
Ahh, the SVG is a 404 not found! That's why. Now just have to figure out which badge it was, out of way too many...

Nevermine, not a problem here (though a better error if a 404 page is returned for the svg would be fantastic).
